compiler-access/src/main/scala/org/alephium/ralph/lsp/access/compiler/ast/node/Node.scala

```diff
@@ -44,45 +50,45 @@ object Node {
  * @param children This node's child nodes.
  * @tparam A Data type.
  */
-case class Node[A] private (
+case class Node[+A, B] private (
```

Comment:
maybe in the doc for `Node` we could explain those 2 parameters? maybe also in the `apply` doc?